### PR TITLE
Fix network section title

### PR DESCRIPTION
**Pull request by bleader**
I'm unsure about the title level needed here, but it seems likely to be 3 just like the `Network objects` part.

> Before submitting the pull request, you must agree with the following statements by checking both boxes with a 'x'.
> * [x] "I accept that my contribution is placed under the CC BY-SA 2.0 license [1]."
> * [x] "My contribution complies with the Developer Certificate of Origin [2]." 
>
> [1] https://creativecommons.org/licenses/by-sa/2.0/
> [2] https://xcp-ng.org/docs/contributing.html#developer-certificate-of-origin-dco

### CODE REVIEW

File: docs/networking.md [hunk 1]
```diff
@@ -29,7 +29,8 @@ This section uses three types of server-side software objects to represent netwo
 * A network, which is a virtual Ethernet switch on a host. Network objects have a name and description, a UUID, and the collection of VIFs and PIFs connected to them.
 
 `xe` CLI, Xen Orchestra or XCP-ng center allow you to configure networking options. You can control the NIC used for management operations, and create advanced networking features such as VLANs and NIC bonds.
-Networks
+
+### Networks
```
Comment (benjamreis) [inline docs/networking.md:33]: I think it should be only `##` as for VLANs, Bonds etc
@stormi what do you think?

Comment (bleader) [reply docs/networking.md:33]: I see, might be, I'll update if we agree on that

**Comment by stormi** [comment]
I think this PR is good as is, but the the whole introduction might need to be improved to make it clearer.

**Review by benjamreis — APPROVED** [review]

**Comment by bleader** [comment]
As discussed with stormi, I added a small improvement in the tip at the top of the page.

**Comment by stormi** [comment]
@olivierlambert will you replicate to the new doc, or do you want a PR to the new doc too?

**Pull request merged**